pages/api/admin/connections/system.ts

```diff
@@ -0,0 +1,29 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
```

Comment:
Would it better if change the API route to `/api/admin/settings/sso`? With this we can add future settings under `/api/admin/settings/[path]` by keeping the same prefix.

Reply:
Yeah that makes sense.